app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java

```diff
@@ -33,6 +33,7 @@
     private boolean isShowInformation;
     private TimeSheetDate mTimeSheetDate;
     private Navigator mNavigator;
+    private boolean isVisibleFloatingActionMenu;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isVisibleFloatingActionMenu has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName